static/js/Misc.jsx

```diff
@@ -1343,9 +1343,17 @@ class CloseButton extends Component {
 class DisplaySettingsButton extends Component {
   render() {
     var style = this.props.placeholder ? {visibility: "hidden"} : {};
-    var icon = Sefaria._siteSettings.TORAH_SPECIFIC ?
-      <img src="/static/img/ayealeph.svg" alt="Toggle Reader Menu Display Settings" style={style} /> :
-      <span className="textIcon">Aa</span>;
+    var icon;
```

Comment:
Use `let` instead of `var`. `var` is from older versions of JS and is no longer very relevant to use.